source/_patterns/01-atoms/icon/scss/_icon.scss

```diff
@@ -8,23 +8,23 @@
 
 // Creates the font face tag for font icons.
 @font-face {
-  font-family: "iconfont";
+  font-family: 'iconfont';
   src: url('../font/iconfont.eot?cachebust=#{random(99999)}'); /* IE9 Compat Modes */
-  src: url('../font/iconfont.eot?#iefix') format('embedded-opentype'), /* IE6-IE8 */
-    url('../font/iconfont.woff') format('woff'), /* Pretty Modern Browsers */
-    url('../font/iconfont.ttf')  format('truetype'), /* Safari, Android, iOS */
-    url('../font/iconfont.svg') format('svg'); /* Legacy iOS */
+  src: url('../font/iconfont.eot?#iefix') format('embedded-opentype'),
```

Comment:
all of the prettier formatting looks fine enough to me, this is the only section that gets annoying because the comments don't line up with the url & formats anymore. //prettier-ignore here?

Reply:
Gah, good catch. These will be things we'll need to watch for, but it's worth it for the other 95% of code. I'll update now.

Reply:
Actually, this gets nuked with @dslemay 's incoming icon work, so we'll let this slide.